src/GUI/Properties/launchSettings.json

```diff
@@ -19,7 +19,7 @@
     "LT.DigitalOffice.GUI": {
       "commandName": "Project",
       "dotnetRunMessages": "true",
-      "launchBrowser": true,
+      "launchBrowser": false,
       "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
       "applicationUrl": "http://localhost:5000",
```

Comment:
maybe we should change 5000 to smth different

Reply:
not need